recipes/keplergl/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "keplergl" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: cb21047b2104413af1c00ef1ac75794a19e0b578e51c69c86713911d97370167
+
+build:
+  noarch: python
+  number: 0
+  script: 
+    - {{ PYTHON }} -m pip install . -vv
+    - jupyter nbextension install --py --sys-prefix keplergl
+
+requirements:
+  host:
+    - python >=3.6
+    - setuptools
+    - pip
+    - jupyter
+    - pandas
+    - geopandas
```

Comment:
You probably don't need `jupyter`, `pandas`, and `geopandas` at host. Unless the package imports itself at build time.

Reply:
Seems that jupyter pandas geopandas are needed at host. Thanks!

Reply:
Sorry for the wild goose chase!

Reply:
Not a problem😁 Thank you very much!